### PR TITLE
{vis}[intel/2015a, intel/2014b] Gnuplot 5.0 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
And updated deps for intel-2015a

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 9 out of 11 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/6d21636b4bea8197e0a6 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/2694848448352d04dc8c for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 2 out of 4 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/bf6114d1998c5f0b3a55 for a full test report.

**Comment by wpoely86** [comment]
Don't merge yet, `cairo` & `intel/2015a` still aren't friends

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/c537546a4687fa3093b6 for a full test report.

**Comment by wpoely86** [comment]
Everything now works!

**Comment by boegel** [comment]
@wpoely86: same of the easyconfigs in here were merged in via #1318, so please resync with develop?

**Comment by wpoely86** [comment]
@boegel done

**Comment by wpoely86** [comment]
@boegel should be good to go

File: easybuild/easyconfigs/c/cairo/cairo-1.12.18-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ('libpng', '1.6.16'),
+    ('freetype', '2.5.5'),
+    ('zlib', '1.2.8'),
+    ('pixman', '0.32.6'),
+    ('fontconfig', '2.11.1'),
+    ('expat', '2.1.0'),
+    ('bzip2', '1.0.6'),
+]
+
+builddependencies = [('Autoconf', '2.69')]
```
Comment (boegel) [inline easybuild/easyconfigs/c/cairo/cairo-1.12.18-intel-2015a.eb]: Autotools?

**Comment by wpoely86** [comment]
@boegel good to go

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f5f9b048f7db46aad1f8 for a full test report.

**Comment by boegel** [comment]
Looking good, thanks @wpoely86!

**Pull request merged**